charts/prom-rds-exporter/Chart.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v2
```

Comment:
helm needs to be published somewhere. We now have only one repo that could build and publish this:
https://github.com/percona/percona-helm-charts